src/android/build.gradle

```diff
@@ -25,7 +25,7 @@ dependencies{
 
     implementation("com.github.outsystems:oscore-android:1.2.0@aar")
     implementation("com.github.outsystems:oscordova-android:2.0.1@aar")
-    implementation("com.github.outsystems:oshealthfitness-android:2.2.1@aar")
+    implementation("com.github.outsystems:oshealthfitness-android:2.3.0-dev1@aar")
```

Comment:
should this be updated with a new version with all the changes in https://github.com/OutSystems/OSHealthFitnessLib-Android/pull/58 ?